components/search_bar/search_bar.jsx

```diff
@@ -155,29 +186,90 @@ export default class SearchBar extends React.Component {
         }
     }
 
+    handleUpdateSearchTerm = (term) => {
+        if (this.state.termsUsed === 0) {
+            this.props.actions.updateSearchTerms(term);
+        } else {
+            const pretextArray = this.props.searchTerms.split(' ');
+            pretextArray.pop();
+            pretextArray.push(term);
+            this.props.actions.updateSearchTerms(pretextArray.join(' '));
+        }
+
+        this.focus();
+        this.setState({index: -1});
+        this.setState({termsUsed: this.state.termsUsed + 1});
+    }
+
+    focus = () => {
+        // This is to allow redux to process the search term change
+        setTimeout(() => {
+            this.search.focus();
+            this.setState({focused: true});
+        }, 0);
+
+        if (this.search.value === '""') {
+            this.search.selectionStart = this.search.length / 2;
+            this.search.selectionEnd = this.search.length / 2;
```

Comment:
In this case, shouldn't `this.search.length / 2` always equal 1? In that case, it would be clearer to just put 1 here. It might also be nice to add a comment here saying that we're putting the selection in between the quotes.